matchzoo/generators/point_generator.py

```diff
@@ -0,0 +1,62 @@
+"""Matchzoo point generator."""
+
+import numpy as np
+from matchzoo import engine
+from matchzoo import tasks
+from matchzoo.datapack import DataPack
+
+
+class PointGenerator(engine.BaseGenerator):
+    """PointGenerator for Matchzoo.
+
+    Ponit generator can be used for classification as well as ranking.
+
+    # Arguments
+        inputs: the transformed dataset
+    """
+
+    def __init__(
+        self,
+        inputs: DataPack,
+        task_type: engine.BaseTask,
```

Comment:
`task` is more appropriate than `task_type` since this parameter is passed as an instance of a task type.

Reply:
ok~